utils/parse.js

```diff
@@ -23,10 +23,10 @@ function keysFromParser(parserPath, parserInstance, parsedResult) {
   if (parsedResult && parsedResult.visitorKeys) {
     return parsedResult.visitorKeys;
   }
-  if (/.*espree.*/.test(parserPath)) {
+  if (typeof parserPath === 'string' && /.*espree.*/.test(parserPath)) {
```

Comment:
if it's not a string wouldn't it fail the test?

Reply:
Because the parserPath would parserOrPath, I added the check to make it more understandable.
If wanted I can remove the check.

Reply:
it's likely this way is more performant and clear, i just wanted to confirm that the checks don't change the behavior